views/login.ejs

```diff
@@ -1,11 +1,28 @@
 <%- include("./partials/header") %>
 
-<h1>Login</h1>
+<div class="container">
+    <div class="row">
+        <div class="col">
+            <h1 style="text-align: center; margin-top: 30px;">Login</h1>
```

Comment:
It's better not to use inline styles in HTML markup. I'd suggest creating a CSS class for styling.
Same about the next line.